lockdown.json

```diff
@@ -0,0 +1,1152 @@
+    "0.2.1": "a1d97ccafcbc2625cc70fadceb36a50c58b01a53"
+  },
+  "arraybuffer.slice": {
+    "0.0.6": "f33b2159f0532a3f3107a272c0ccfbd1ad2979ca"
+  },
+  "arrify": {
+    "1.0.0": "d6c361518250802fa2147ea7fb67597128cb8c81"
+  },
+  "asap": {
+    "1.0.0": "b2a45da5fdfa20b0496fc3768cc27c12fa916a7d",
+    "2.0.3": "1fc1d1564ee11620dfca6d67029850913f9f4679"
+  },
+  "asn1": {
+    "0.1.11": "559be18376d08a4ec4dbe80877d27818639b2df7"
+  },
+  "assert-plus": {
+    "0.1.5": "ee74009413002d84cec7219c6ac811812e723160"
+  },
+  "async": {
+    "0.1.22": "0fc1aaa088a0e3ef0ebe2d8831bab0dcf8845061",
+    "0.2.10": "b6bbe0b0674b9d719708ca38de8c237cb526c3d1",
+    "0.9.2": "aea74d5e61c1f899613bf64bda66d4c78f2fd17d",
+    "1.4.2": "6c9edcb11ced4f0dd2f2d40db0d49a109c088aab"
+  },
+  "async-each": {
+    "0.1.6": "b67e99edcddf96541e44af56290cd7d5c6e70439"
+  },
+  "aws-sign2": {
+    "0.5.0": "c57103f7a17fc037f02d7c2e64b602ea223f7d63"
+  },
+  "backo2": {
+    "1.0.2": "31ab1ac8b129363463e35b3ebb69f4dfcfba7947"
+  },
+  "balanced-match": {
+    "0.2.0": "38f6730c03aab6d5edbb52bd934885e756d71674"
+  },
+  "base64-arraybuffer": {
+    "0.1.2": "474df4a9f2da24e05df3158c3b1db3c3cd46a154"
+  },
+  "base64id": {
+    "0.1.0": "02ce0fdeee0cef4f40080e1e73e834f0b1bfce3f"
+  },
+  "benchmark": {
+    "1.0.0": "2f1e2fa4c359f11122aa183082218e957e390c73"
+  },
+  "better-assert": {
+    "1.0.2": "40866b9e1b9e0b55b481894311e68faffaebc522"
+  },
+  "binary-extensions": {
+    "1.3.1": "32dd9ed2a7c69acec56f77f6cd80df043f78777a"
+  },
+  "bindings": {
+    "1.2.1": "14ad6113812d2d37d72e67b4cacb4bb726505f11"
+  },
+  "bl": {
+    "0.9.4": "4702ddf72fbe0ecd82787c00c113aea1935ad0e7",
+    "1.0.0": "ada9a8a89a6d7ac60862f7dec7db207873e0c3f5"
+  },
+  "blob": {
+    "0.0.4": "bcf13052ca54463f30f9fc7e95b9a47630a94921"
+  },
+  "bluebird": {
+    "2.10.2": "024a5517295308857f14f91f1106fc3b555f446b"
+  },
+  "body-parser": {
+    "1.14.1": "ffe921eba3ce8f191e2a8a8803844bd025f3c6dc"
+  },
+  "boom": {
+    "0.4.2": "7a636e9ded4efcefb19cef4947a3c67dfaee911b",
+    "2.9.0": "a54b7fd2fee477d351bf9e371680cbea67f12715"
+  },
+  "brace-expansion": {
+    "1.1.1": "da5fb78aef4c44c9e4acf525064fb3208ebab045"
+  },
+  "braces": {
+    "0.1.5": "c085711085291d8b75fdd74eab0f8597280711e6",
+    "1.8.1": "2d195b85a0a997ec21be78a7f1bc970480b12a1a"
+  },
+  "bufferutil": {
+    "1.2.1": "37be5d36e1e06492221e68d474b1ac58e510cbd7"
+  },
+  "builtin-modules": {
+    "1.1.0": "1053955fd994a5746e525e4ac717b81caf07491c"
+  },
+  "bytes": {
+    "2.1.0": "ac93c410e2ffc9cc7cf4b464b38289067f5e47b4"
+  },
+  "callsite": {
+    "1.0.0": "280398e5d664bd74038b6f0905153e6e8af1bc20"
+  },
+  "camelcase": {
+    "1.2.1": "9bb5304d2e0b56698b2c758b08a3eaa9daa58a39"
+  },
+  "camelcase-keys": {
+    "1.0.0": "bd1a11bf9b31a1ce493493a930de1a0baf4ad7ec"
+  },
+  "caseless": {
+    "0.11.0": "715b96ea9841593cc33067923f5ec60ebda4f7d7",
+    "0.6.0": "8167c1ab8397fb5bb95f96d28e5a81c50f247ac4"
+  },
+  "casperjs": {
```

Comment:
We just recently removed this from `package.json` (along with phantomJS), so it might be worth updating the lockdown file here, as they are no longer needed :)